Feature/gh 5 the projects readme

Pylyv opened the pull request:
Requesting PR of the Issue #5 for the changes of the README performed once it's possible and it doesn't cause conflicts in the main branch

Pylyv committed:
feat: Adding CONTRIBUTING file to move forward with the README file on the Rumble-LoL-Plugin project

Pylyv committed:
Merge remote-tracking branch 'origin/main' into feature/GH-5-the-projects-readme

Pylyv committed:
feat: Resolving some TODOs from the README file for the issue #5  "The project's README"

Pyzyryab commented:
Hello.

For the moment, we still need to finish the base documentation before bring your improvement changes over it.

I'll finish those changes ASAP, so you can merge them back into the changes of this PR

Pylyv committed:
feat: Added status badges
feat: Merge remote-tracking branch 'origin/main' into feature/GH-5-the-projects-readme

Pylyv committed:
feat: Added status badges
feat: Merge remote-tracking branch 'origin/main' into feature/GH-5-the-projects-readme
